### PR TITLE
Develop Production Config

### PR DESCRIPTION
**Pull request by imakshayverma**
- Split settings files.

### CODE REVIEW

File: union_budget_board/settings/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .development import *
```
Comment (heaven00) [inline union_budget_board/settings/__init__.py:1]: sorry, just a question. Why is this required here ?

File: union_budget_board/settings/production.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+from .base import *
```
Comment (heaven00) [inline union_budget_board/settings/production.py:1]: can we change it to constants that are necessary only rather then `*`

Incase these are too many then we can use `import *`

File: union_budget_board/settings/__init__.py (hunk 1 above)
Comment (imakshayverma) [reply union_budget_board/settings/__init__.py:1]: So we start the server, this is the development.py is used by default. For Production one to be used we would need to mention in the suitable environment variable.

File: union_budget_board/settings/production.py (hunk 2 above)
Comment (imakshayverma) [reply union_budget_board/settings/production.py:1]: base.py has the basic settings for both development and production environment. It has the minimal settings that we need to have. The ones that are different for development and prod config, would be mentioned in their respective files.

File: union_budget_board/settings/__init__.py (hunk 1 above)
Comment (heaven00) [reply union_budget_board/settings/__init__.py:1]: okay, but generally thats part of the run.py parameters which one you want to run. But lets go with this for now.

File: union_budget_board/settings/production.py (hunk 2 above)
Comment (heaven00) [reply union_budget_board/settings/production.py:1]: okay, then can we remove `ALLOWED_HOSTS` and `DEBUG` from base.py ?

File: union_budget_board/settings/__init__.py (hunk 1 above)
Comment (imakshayverma) [reply union_budget_board/settings/__init__.py:1]: There are multiple ways to split file - [Link](https://code.djangoproject.com/wiki/SplitSettings).  This way we don't have to give parameter for which environment we have to run it in.

File: union_budget_board/settings/production.py (hunk 2 above)
Comment (imakshayverma) [reply union_budget_board/settings/production.py:1]: Yes. We can.

Comment (heaven00) [reply union_budget_board/settings/production.py:1]: :+1: just that change and we are good to merge

**Review by heaven00 — APPROVED** [review]
